hardware_interface/test/test_component_interfaces.cpp

```diff
@@ -351,29 +360,29 @@ using hardware_interface::hardware_interfaces_components_test::DummySystemHardwa
 class TestComponentInterfaces : public Test
 {
 protected:
-  ComponentInfo joint_info;
-  ComponentInfo sensor_info;
+  ComponentInfo joint_info_;
+  ComponentInfo sensor_info_;
 
   void SetUp() override
   {
-    joint_info.name = "DummyPositionJoint";
-    joint_info.parameters["max_position"] = "3.14";
-    joint_info.parameters["min_position"] = "-3.14";
+    joint_info_.name = "DummyPositionJoint";
+    joint_info_.parameters["max_position"] = "3.14";
+    joint_info_.parameters["min_position"] = "-3.14";
```

Comment:
```suggestion
    joint_info_.parameters["max"] = "3.14";
    joint_info_.parameters["min"] = "-3.14";
```

Reply:
I believe the "parameters" string here is very flexible. It can be anything, right? With that said, I made this change.